### PR TITLE
Issue #1439: EL6 uses Python2.7

### PR DESCRIPTION
**Pull request by tst-cjeffries**
Changes EL6 to use Python2.7 from the SCL repositories. El7/Ubuntu are unaffected. 

To test:

- On all OS (RHEL6/7, CentOS6/7, Ubuntu 16):

- Build and install OpenGEE (trying both the scripts and the RPMS)

- Test python elements (snippets, WMS, all the various searches)

- Build and publish the tutorial data.

### CODE REVIEW

**Comment by tst-dsugar** [comment]
The travis issue seems unrelated and this is a branch in progress.

E: Failed to fetch http://us-east-1.ec2.archive.ubuntu.com/ubuntu/dists/trusty-updates/main/source/Sources.gz  Hash Sum mismatch
E: Some index files failed to download. They have been ignored, or old ones used instead.
The command "sudo apt-get update -qq" failed and exited with 100 during .

**Comment by tst-cjeffries** [comment]
This isn't quite ready for merge yet. I have to make a few small changes on how the platform is detected for setting the python version.

File: earth_enterprise/src/third_party/numpy/SConscript [hunk 1]
```diff
@@ -52,7 +52,7 @@ numpy_extract = numpy_env.Command(
                        numpy_target))])
 
 # [2] Build numpy.
-python_cmd = '%s/bin/python' % root_dir
+python_cmd = numpy_env['python_bin']
```
Comment (tst-dsugar) [inline earth_enterprise/src/third_party/numpy/SConscript]: so this picks up the python bin path from numpy?  I assume numpy detects and sets it from somewhere...

Comment (tst-cjeffries) [reply earth_enterprise/src/third_party/numpy/SConscript]: This is a variable copied from the parent scons tasks. The numpy_env in this case is a variable containing the build environment.

**Pull request merged**